__tests__/app.test.js

```diff
@@ -45,6 +45,29 @@ describe("GET /api/", () => {
   });
 });
 
+describe("GET /api/articles", () => {
+  it("200: should respond with an array of all articles in descending order by date", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSorted("created_at", { descending: true });
```

Comment:
This is a really good test, love the use of the sorted package here to check these are coming back in the correct order.